ethereumkit/src/test/java/org/p2p/ethereumkit/EthereumKeyPairGeneratorTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.p2p.ethereumkit
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.p2p.ethereumkit.internal.core.EthereumKit
+import org.p2p.ethereumkit.external.repository.EthereumKitRepository
+
+private const val ETHEREUM_ADDRESS_LENGTH = 42
+
+class EthereumKeyPairGeneratorTest {
+    //This seed phrase was taken from ethereum kit repository, so feel free to use it
+    private val seedPhrase = "apart approve black comfort steel spin real renew tone primary key cherry".split(" ")
+
+    @Before
+    fun setup() {
+        EthereumKit.init()
```

Comment:
if we need to launch this thing when app starts - better to place it in repository now